onnxruntime/python/tools/quantization/calibrate.py

```diff
@@ -345,25 +345,31 @@ def calibrate(model_path,
               op_types=['Conv', 'MatMul'],
               black_nodes=[],
               white_nodes=[],
-              augmented_model_path='augmented_model.onnx'):
+              augmented_model_path='augmented_model.onnx',
+              providers=["CPUExecutionProvider"],
+              tensorrt_calibration=False):
```

Comment:
couldn't any tensorrt required logic be encapsulated in the caller and not here?
let's sync offline.